app/services/peaks/processor.rb

```diff
@@ -27,7 +27,8 @@ def generate(remote_file)
 
       peaks
     ensure
-      system("rm -rf #{raw_path.gsub('/audio.wave', '')}") if raw_path
+      sanitized_raw_path = Shellwords.escape(raw_path.gsub('/audio.wave', ''))
```

Comment:
I consulted with ChatGPT for this solution. If there is a better more eloquent solution please let me know! I was having a hard time finding via google.

Reply:
Brakeman Audit:
Confidence: Medium
Category: Command Injection
Check: Execute
Message: Possible command injection
Code: system("rm -rf #{raw_path.gsub("/audio.wave", "")}")
File: app/services/peaks/processor.rb
Line: 29